### DISCUSSION TITLE
Editor timeline looks polluted with the current way volume, sampleset and sv changes are displayed

### DISCUSSION THREAD

**Discussion by maotovisk** [ppy/osu#17800] [Ideas] [answered]
As a mapper, in my opinion, it looks polluted to display it individually for each hitobject. 
It would make more sense to display only the changes from sampleset, volume and slider velocity in the timeline. Even though now the logic from each of those attributtes is tied to each hitobject individually. 

![image](https://user-images.githubusercontent.com/20617307/163078229-bed3240d-8547-4c2c-a124-ffb2ba117b2e.png)

**Comment by peppy** [comment, marked as answer]
Making redundant points like that will become a less common thing going forward. The idea is that most changes which are "per object" should be attached to the object rather than the timeline.

The display will be changed as we move forward to avoid this kind of overlap.

**Reply by maotovisk** [reply]
Moving the display to outside the timeline seems to be a good start for it.

**Reply by peppy** [reply]
If you mean further up or down, that's how it used to be and people complained it was too big.

You can also choose to turn off the display for now if you'd prefer.

**Reply by maotovisk** [reply]
I meant outside the timeline as in another place like a toolbox in the right, for example.

**Reply by peppy** [reply]
If you want this, you can *turn the display off* on the timeline.

Per hitobject properties will be accessible to the right (this has been planned from the outset).
